### DISCUSSION TITLE
For Galaxy Brain achievement, does it have to be in Q&A?

### DISCUSSION THREAD

**Discussion by ghost** [Schweinepriester/github-profile-achievements#13] [Q&A] [answered]
Anyone knows?

**Comment by melvincwng** [comment, marked as answer]
Yes, from what I see here, it should be in Q&A to count for 'Galaxy Brain' achievement:
- https://github.com/Schweinepriester/github-profile-achievements/discussions/9
- https://github.com/Schweinepriester/github-profile-achievements/discussions/10

But, to be honest, I'm not entirely sure of the full criteria that's actually needed to unlock the achievement. 

Seems like other developers in this community (as seen from the 'Discussions' tab) are still trying to figure out and understand the full criteria needed to unlock this 'Galaxy Brain' achievement. 

Perhaps, you can check back this page/repository again for any updates.

**Comment by deniiiiz-i** [comment]
Yes it's necessary

**Comment by th3qui85ly** [comment]
How to exactly count up as marked one basically?

**Comment by uiolee** [comment]
test
